examples/prometheus/prometheus.yaml

```diff
@@ -643,6 +647,66 @@ objects:
           action: replace
           target_label: kubernetes_name
 
+      # Scrape aggregated logging endpoints.
+      #
+      # The relabeling allows the actual service scrape endpoint to be configured
+      # via the following annotations:
+      #
+      # * `prometheus.io/scrape`: Only scrape services that have a value of `true`
+      # * `prometheus.io/scheme`: If the metrics endpoint is secured then you will need
+      # to set this to `https` & most likely set the `tls_config` of the scrape config.
+      # * `prometheus.io/path`: If the metrics path is not `/metrics` override this.
+      # * `prometheus.io/port`: If the metrics are exposed on a different port to the
+      # service then set this appropriately.
+      - job_name: 'kubernetes-logging-service-endpoints'
+
+        tls_config:
+          ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+          # TODO: this should be per target
+          insecure_skip_verify: true
+        bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+
+        kubernetes_sd_configs:
+        - role: endpoints
+          namespaces:
+            names:
+            - 'openshift-logging'
+
+        relabel_configs:
+          # only scrape infrastructure components
+          - source_labels: [__meta_kubernetes_namespace]
+            action: keep
+            regex: 'default|logging|metrics|kube-.+|openshift|openshift-.+'
```

Comment:
Why is this value different than the above?

Reply:
Yes, this seems incorrect now. Will push update.